CONTRIBUTING.md

```diff
@@ -199,7 +199,7 @@ When using OpenShift, the OLM is already installed.
 
 ### Create the bundle and related images
 
-The following commands will generate a bundle under `bundle/`, build an image with its contents, build and publish the operator image.
+The following commands will generate two bundles (one for regular Kubernetes and another one for OpenShift) under `bundle/`, build an image with its contents, build and publish the operator image.
 
 ```bash
 BUNDLE_IMG=docker.io/${USER}/opentelemetry-operator-bundle:latest IMG=docker.io/${USER}/opentelemetry-operator:latest make bundle container container-push bundle-build bundle-push
```

Comment:
Which bundle image does this command build? The one for OCP or community? Should it build two?

Reply:
We are not using the OpenShift bundle image for anything. Do we need to push it somewhere?
Previously, we published the image for plain Kubernetes because we were planning to use it for the E2E tests but it is not needed for anything else. I think we can just skip until somebody asks for it.